.gitattributes

```diff
@@ -1,4 +1,2 @@
 compiler-tests/**/*.java linguist-generated=true
-compiler-tests/**/*.fir.txt linguist-generated=true
-compiler-tests/**/*.fir.diag.txt linguist-generated=true
```

Comment:
Curious why you removed these, do you think they should be non-collapsed since they should be reviewed?

Reply:
Yes, I believe they should be reviewed. While they are initially generated by the test framework, if there is a change from an existing file, it is considered a test failure. Sort of like a golden value test.

The diag files are essentially the same as the assertions you had for the error/warning messages. And the fir files are essentially the same as checking for generated classes/functions in the class files.